templates/resources/kustomization.md.tmpl

```diff
@@ -0,0 +1,39 @@
+---
+Title: "Kustomization Resource"
+Description: |-
+    Creating the Kustomization resource.
+---
+
+# Kustomization
+
+The `tanzu-mission-control_kustomization` resource allows you to add, update, and delete Kustomization to a particular scope through Tanzu Mission Control.
+
+To create a kustomization, you must be associated with the cluster.admin or clustergroup.admin role
+
+In Creation of kustomization we must required to create Git Repository first, which we need to referenced in spec of kustomization, Git Repository can be created by using "tanzu-mission-control_git_repository" resource from terraform provider itself.
+
+[kustomization]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-99916A6D-5DAF-4A26-88C7-28662F847F2F.html
+
+## kustomization Scope
```

Comment:
Is this the right heading, please check. Also, `Kustomization`

Reply:
Updated